src/data-access/user.js

```diff
@@ -0,0 +1,67 @@
+import { Op } from 'sequelize';
+import sequelize from '../db/db.js';
+import User from '../models/user.js';
+
+async function createUser(login, password, age) {
+  const user = await User.create({ login, password, age });
+
+  return user;
+}
+
+async function updateUser(id, login, password, age) {
+  const user = await User.update(
```

Comment:
please add appropriate error handling in case if some error occurred during communication with database. As a client I expect to receive 500 status code Internal Error. 
Consider throwing custom errors on data access layer, like `throw new DataAccessException` and handling them in express error middleware

https://expressjs.com/en/guide/error-handling.html

Reply:
Done